cmd/generate/runner.go

```diff
@@ -66,14 +65,11 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 	}
 
 	in := generator.GenerateInput{
-		App:           r.flag.App,
-		ConfigVersion: r.flag.ConfigVersion,
-
+		App:       r.flag.App,
 		Name:      r.flag.Name,
 		Namespace: r.flag.Namespace,
 
 		ExtraAnnotations: map[string]string{
-			meta.Annotation.ConfigVersion.Key():   r.flag.ConfigVersion,
```

Comment:
Don't we want to set the output of `git describe --tags` here?

Reply:
This change will go here: https://github.com/giantswarm/konfigure/pull/3